packages/genesys-spark-components/src/components/beta/gux-rich-text-editor/gux-rich-text-editor-beta.tsx

```diff
@@ -309,6 +326,34 @@ export class GuxRichTextEditor {
     ].some(child => child !== null && child !== undefined);
   }
 
+  private setupDragAndDrop(): void {
+    const editorElement = this.root?.querySelector('[slot="editor"]');
+    if (editorElement) {
+      editorElement.addEventListener('dragover', (e: DragEvent) => {
+        e.preventDefault();
+        this.showDropOverlay = true;
+      });
+
+      editorElement.addEventListener('dragleave', (e: DragEvent) => {
+        if (!editorElement.contains(e.relatedTarget as Node)) {
+          this.showDropOverlay = false;
+        }
+      });
+
+      this.root.addEventListener('filesDropped', (e: CustomEvent) => {
```

Comment:
The reason for not using `drop` on the editorElement was that Tiptaps editor enviroment was actually intercepting the events from what I could tell so `drop` event would never execute. Let me know if you see anything wrong with this approach.